src/jdk.dynalink/share/classes/jdk/dynalink/SecureLookupSupplier.java

```diff
@@ -39,6 +39,11 @@
  * @since 9
  */
 public class SecureLookupSupplier {
+    /**
+     * The name of a runtime permission required to successfully invoke the
+     * {@link #getLookup()} method.
+     */
+    public static final String GET_LOOKUP_PERMISSION_NAME = "dynalink.getLookup";
```

Comment:
`AUTOLOAD_PERMISSION_NAME` below has an `@apiNote` that would be worth replicating here as well.

Reply:
As @AlanBateman pointed out, the class has an API note so the member doesn't need one separately.